<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QPushButton" name="pbUcitajSastojke">
     <property name="text">
      <string>Ucitaj sastojke</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Kupovina</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Broj kupaca</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leBrojKupaca"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pbZapocniKupovinu">
       <property name="text">
        <string>Zapocni kupovinu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="teIspis"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
